pom.xml

```diff
@@ -23,19 +23,19 @@
     <dependency>
       <groupId>org.graalvm.sdk</groupId>
       <artifactId>graal-sdk</artifactId>
-      <version>1.0.0-rc9</version>
+      <version>19.1.1</version>
```

Comment:
Nit: put the graal version into the properties and use that rather than the hardcoded one.